components/announcement_bar/overage_users_banner/index.tsx

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+import {FormattedMessage, useIntl} from 'react-intl';
+import {useDispatch, useSelector} from 'react-redux';
+
+import {getCurrentUser, isCurrentUserSystemAdmin} from 'mattermost-redux/selectors/entities/users';
+import {GlobalState} from 'types/store';
+import {getLicense} from 'mattermost-redux/selectors/entities/general';
+import AnnouncementBar from 'components/announcement_bar/default_announcement_bar';
+import {calculateOverageUserActivated} from 'utils/overage_team';
```

Comment:
Is this a new file? Doesn't seem to be part of the PR?

Reply:
Yes, but this PR is created from another branch. https://github.com/mattermost/mattermost-webapp/pull/11482/files#diff-d01376ffc6b70925fed81daa8f355304ae72969f3923a6227017572c1b24f6f2R2185